homeassistant/components/device_tracker/__init__.py

```diff
@@ -467,10 +473,9 @@ def update_config(path: str, dev_id: str, device: Device):
 
         for key, value in (('name', device.name), ('mac', device.mac),
```

Comment:
I think that it would make more sense of instead writing our own YAML, we do `out.write(yaml.dumps(data))`